### ISSUE TITLE
Can medusa be installed on an admin node?

### ISSUE THREAD

**Issue by chrisjmiller1** [thelastpickle/cassandra-medusa#769]
Hi folks, 

Is it possible to install Medusa on an admin node i.e. a node that is not part of the target cassandra cluster as I know the cassandra.yaml location must be specified.

Thanks, 

Chris.

**Comment by rzvoncek** [comment]
Hi @chrisjmiller1 !

This might actually work to some extent. You will have to grab the `cassandra.yaml` file and place it on the _admin_ node. Medusa learns some things from this, for example the `listen_address` of the node. If this one is reachable from the admin node, Medusa running on the admin node will connect to it as if it were a local connection. 

The first thing that will break when doing a `medusa backup` is the file upload. This is because we assume the files are locally available and pass local paths as arguments to the upload methods.
Doing a `medusa backup-cluster` from the admin node has a better chance of suceeding because there the admin node ssh-s into all the Cassandra nodes and runs itself there. But this still requires you to install Medusa on the C* nodes.

Another important thing you should configure is the `storage/fqdn` (or use `--fqdn` CLI option). This will allow the admin node to "impersonate" the Cassandra node when interacting with storage.

So in conclusion, making backups without installing Medusa on Cassandra nodes will not work. Listing, downloading or restoring the backups has a bigger chance of success.

**Comment by chrisjmiller1** [comment]
Thanks @rzvoncek I will look into this and will let you know how it goes.
